observability/metrics/src/main/scala/com/daml/metrics/MetricsConfig.scala

```diff
@@ -8,7 +8,12 @@ import pureconfig.{ConfigReader, ConvertHelpers}
 import pureconfig.generic.semiauto.deriveReader
 import scala.concurrent.duration._
 
-final case class MetricsConfig(reporter: MetricsReporter, reportingInterval: FiniteDuration)
+final case class HistogramDefinition(nameRegex: String, buckets: Seq[Double])
```

Comment:
I'd recommend making the `buckets` parameter name here a bit more specific, e.g., maybe call it `bucketBoundaries` as it describes the boundaries of the buckets that we're defining for this histogram?

Reply:
Sure, will change